models/app.model.js

```diff
@@ -0,0 +1,59 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db
+    .query("SELECT slug, description FROM topics;")
+    .then((res) => {
+      return res.rows;
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
We don't need a catch block in here, because any errors that come from the model will automatically make our controller promises fail. That being said, please do some error handling in your controllers.